packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -86,6 +86,38 @@ def self.set_node_modules_user_settings(installer, react_native_path)
         end
     end
 
+    def self.set_ccache_compiler_and_linker_build_settings(installer, react_native_path, ccache_enabled)
+        projects = self.extract_projects(installer)
+
+        ccache_path = `command -v ccache`.strip
+        ccache_available = !ccache_path.empty?
+
+        message_prefix = ccache_available ? "⚡️".yellow + "Ccache detected (found at %s): " % [ccache_path.italic] : "Unable to locate ccache: "
```

Comment:
this prefix can become quite long, as it depends on the path found at line 92.
What about just have `[CCACHE]` as a general prefix and have a first line of log that is
```
Pod::UI.puts("#{message_prefix}: Ccache found at #{ccache_path}")
```
?

Reply:
Also, I prefer using string interpolation rather than formatting. It feels more Swift/iOS and it is easier to read as the symbols are already replaced by their variable.

Reply:
Makes sense 👍 I've updated the code to use string interpolation and a `[Ccache]` prefix.